lib/schemas/schemaTranslations.json

```diff
@@ -16,6 +16,31 @@
         "^[a-z]{2}$": {
           "type": "string",
           "minLength": 1
+        },
+        "^[-a-zA-Z]{5}$": {
```

Comment:
What you thought about make it more specific with `^[a-z]{2}-[A-Z]{2}$`(just for the 5 character one)  or `^[a-z]{2}(-[A-Z]{2})?$` which would validate both 2 character and the 5 character one.